### DISCUSSION TITLE
hello, has any one tried to run kvrocks on K8s?

### DISCUSSION THREAD

**Discussion by Goutami-Gowda** [apache/kvrocks#1176] [General]
hello, has any one tried to run kvrocks on K8s?

**Comment by tisonkun** [comment]
@Goutami-Gowda Since Kvrocks provides an official image, you should deploy Kvrocks on k8s as easily as Redis, except that you may take care of binding persistent volume as Kvrocks persists data.

**Reply by Goutami-Gowda** [reply]
yes i tried deploying kvrocks as daemon set with persistent storage type has host path. but i am not able to bring up kvrocks cluster with 2 master pods. as they are not able to communicate with each other.
